docs/tests-development.md

```diff
@@ -0,0 +1,16 @@
+## Writing tests
+
+When writing tests please follow the same approach as we have taken in the other tests. There are many ways to 
+test software and we have chosen ours, so please appreciate that.
+
+The main tests are provided in `test` folder. Before execute them, first make sure that the Keycloak server was started to run all the integration tests:
```

Comment:
I believe there should be "Before ***executing*** them". But my English is far from perfection so I might be wrong. :)

Reply:
@vmuzikar you're correct. Thank you.

Reply:
@abstractj Just a silly question but have you pushed your changes already? :) The file seems unchanged.